crates/bevy_ptr/src/lib.rs

```diff
@@ -208,7 +208,7 @@ impl<'a, T> ThinSlicePtr<'a, T> {
     /// Indexes the slice without doing bounds checks
     ///
     /// # Safety
-    /// `index` must be inbounds.
+    /// `index` must be in-bounds.
```

Comment:
I think it was missing [here](https://github.com/bevyengine/bevy/blob/4b1f6f4ebba92fc968124380d5a19e1d2500e2dd/crates/bevy_ecs/src/storage/blob_vec.rs#L320)